### DISCUSSION TITLE
Hi, let's talk.

### DISCUSSION THREAD

**Discussion by steviegt6** [steviegt6/holocure-modloader#10] [Announcements]
I noticed some traffic coming to the repo lately, so I felt that it may be appropriate to open a place of discussion for any of those interested in the project.

**Comment by RCoder01** [comment]
Hello!
I'm interested in looking at the internals of HoloCure, but not very proficient with C# and certainly not knowledgeable enough to write a full application just to be able to read the HoloCure source. Other tools like UndertaleModTool weren't working for me so I was hoping this specifically designed program would work, I'm just not very sure how to use it. I got to the point of successfully installing and building but am not sure how to proceed. Any help would be appreciated!

**Reply by steviegt6** [reply]
You can use [DogScepter](https://github.com/colinator27/DogScepter) for the time being. Their CLI app supports dumping decompiled code, but not recompliation yet.

**Reply by RCoder01** [reply]
Thanks for the tip! That seems to be working for the most part. It says it can't decompile "gml_GlobalScript_shell_scripts" but that seems to only be one script.

**Reply by steviegt6** [reply]
Ah yeah, I had issues decompiling that one. I believe the [latest build of UMT](https://github.com/krzys-h/UndertaleModTool/suites/7441529633/artifacts/304290913) can decompile that one, though.

**Reply by RCoder01** [reply]
That works! Thanks!

**Reply by steviegt6** [reply]
No problem, glad to be of help.

**Comment by yeldarbchaotic** [comment]
Hi, I'm not really used to git or github, can you tell me why I get an error saying "Can't find git.targets" when building with dotnet? The file definitely doesn't exist, but I'm wondering if I'm supposed to get it from somewhere else, or if it's supposed to be in here already and just isn't? Sorry if the solution is obvious, I just can't figure it out. Thanks!

**Reply by RCoder01** [reply]
I had this same issue too. I think my issue was that I was using an old version of node.js which for some reason didn't execute the build process entirely.

**Reply by steviegt6** [reply]
Batch script is broken, last I checked. Don't bother using this tool for now anyway, it needs to mature a bit more (some day...).
